introduction.md

```diff
@@ -1 +1,2 @@
-My name is James Donaldson and I am a JavaScript developer working at Starbucks.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?